### ISSUE TITLE
[$15][MSFT-16] [Challenge Feed] Element is in tab order and does not have accessible text

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2740]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/challenges and Login using valid credentials
2. Right-click > Inspect Elements 
3. Check the Logo code

**Expected Result**
- Element does not have text that is visible to screen readers
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="presentation"
- Element's default semantics were not overridden with role="none"

**Actual Result**
Ensures links have discernible text

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 2. Operable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.4 Link Purpose (In Context)
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=58da9b68%2Df2a3%2D45d2%2Dbbae%2D15abdfd0865b&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2Fcodejam%2DAccessibility%2F2

**Comment by ThomasKranitsas** [comment]
Contest https://www.topcoder.com/challenges/30096123 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for thomaskranitsas via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096123 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by sushilshinde** [comment]
Wait for 15 mins

**Comment by lakshmiathreya** [comment]
<img width="1507" alt="Screenshot 2019-07-22 at 4 23 48 PM" src="https://user-images.githubusercontent.com/51733042/61627347-27dabb80-ac9d-11e9-8694-5306557e5d32.png">
Aria attributes not set ...

**Comment by codeMinter** [comment]
@r0hit-gupta we need to make the change for Mobile Header too. Can you add PR for the mobile header too?

**Comment by veshu** [comment]
@r0hit-gupta Please open separate PR for the mobile, so that we don't have revert the changes

**Comment by codeMinter** [comment]
@r0hit-gupta we have related issues to this, can you help get this done ASAP ?

**Comment by r0hit-gupta** [comment]
@codeMinter working on it

**Comment by r0hit-gupta** [comment]
@codeMinter please review.

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097421 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097421 has been updated - it has been assigned to rohitgupta_.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by tosha5252** [comment]
Verified 

Env: Windows 10
Browser: Chrome and Edge

**Comment by tosha5252** [comment]
Checked using WAVE tool in Chrome Browser...no aria label issues...verified passing

**Comment by tosha5252** [comment]
OS: Windows 10

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097421<br/><br/>```This is an automated message for crazyk via Topcoder X```
